### PR TITLE
chore: fixed projen deps

### PR DESCRIPTION
**Pull request by voho**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,13 +63,13 @@
     "typescript": "^4.5.5"
   },
   "peerDependencies": {
-    "@aws-cdk/core": "^1.123.0",
+    "@aws-cdk/core": "1.123.0",
     "constructs": "^3.2.27",
-    "monocdk": "^1.123.0"
+    "monocdk": "1.123.0"
```
[the remark this answers is not in the document]
Comment (ayush987goyal) [reply package.json]: @voho might also wanna remove the core dependency since we are using monocdk?

[the remark this answers is not in the document]
Comment (voho) [reply package.json]: That should be removed now - can you check?

File: package.json [hunk 2]
```diff
@@ -61,13 +63,13 @@
     "typescript": "^4.5.5"
   },
   "peerDependencies": {
-    "@aws-cdk/core": "^1.123.0",
+    "@aws-cdk/core": "1.123.0",
```
[the remark this answers is not in the document]
Comment (voho) [reply package.json]: We could also ask Elad about this.

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (ayush987goyal) [reply package.json]: I can still see `@aws-cdk/core` in deps and peer-deps.

**Pull request closed without merging**